galore/__init__.py

```diff
@@ -189,13 +206,14 @@ def process_pdos(input=['vasprun.xml'],
                 assert data.dtype.names[0] == energy_label
             except AssertionError as error:
                 error.args += ("Energy labels are not consistent "
-                               "between input files",)
+                               "between input files", )
                 raise
 
         orbital_labels = data.dtype.names[1:]
         pdos_data[element] = OrderedDict([('energy', data[energy_label])])
-        pdos_data[element].update(OrderedDict((orbital, data[orbital])
-                                  for orbital in orbital_labels))
+        pdos_data[element].update(
+            OrderedDict(
+                (orbital, data[orbital]) for orbital in orbital_labels))
```

Comment:
It's difficult as a reviewer to see the real changes when they are mixed in with so many unnecessary style changes. The existing style was already PEP8-compliant, these indentation changers are not necessary.

It doesn't mean they're _bad_, but it would help if these style changes were a separate PR.